cartridges/int_alma/cartridge/scripts/helpers/almaPlanHelper.js

```diff
@@ -135,6 +135,7 @@ function getPlansForCheckout(locale, currentBasket) {
     });
 
     plans = almaUtilsHelpers.map(plans, function (plan) {
+        plan.purchaseAmount = purchaseAmount; // eslint-disable-line no-param-reassign
```

Comment:
Are you sure that remove `Math.round()` bellow will not make problems here ?

Reply:
yes :), as discussed together we should remove it here and in the other place too